### PR TITLE
Fix proxy_get defined in riak-cs.conf not working [JIRA: RCS-236]

### PR DESCRIPTION
**Pull request by kuenishi**
Without this code you'll have excessive log output like this by default:

```
2015-07-21 18:35:34.842 [warning] <0.814.0>@riak_cs_config:proxy_get_active:289 proxy_get value in app.config is invalid
2015-07-21 18:35:35.949 [warning] <0.819.0>@riak_cs_config:proxy_get_active:289 proxy_get value in app.config is invalid
2015-07-21 18:35:37.534 [warning] <0.824.0>@riak_cs_config:proxy_get_active:289 proxy_get value in app.config is invalid
2015-07-21 18:35:38.381 [warning] <0.829.0>@riak_cs_config:proxy_get_active:289 proxy_get value in app.config is invalid
2015-07-21 18:35:38.957 [warning] <0.834.0>@riak_cs_config:proxy_get_active:289 proxy_get value in app.config is invalid
2015-07-21 18:35:39.834 [warning] <0.839.0>@riak_cs_config:proxy_get_active:289 proxy_get value in app.config is invalid
```

This is because `riak_cs_config:proxy_get_active/0` handles `proxy_get_active` config item in riak-cs.config badly, only accepting `enabled/disabled` atom which could be set at `advanced.config`.

### CODE REVIEW

File: src/riak_cs_config.erl [hunk 1]
```diff
@@ -277,16 +277,16 @@ set_md5_chunk_size(_) ->
...
-        {ok, _} ->
-            _ = lager:warning("proxy_get value in app.config is invalid"),
-            false;
-        undefined -> false
+    case application:get_env(riak_cs, proxy_get, false) of
+        enabled ->   true;
+        disabled -> false;
+        Flag when is_boolean(Flag) -> Flag;
+        Other ->
+            _ = lager:warning("proxy_get value in riak.conf (advanced.config) is invalid: ~p",
```
Comment (shino) [inline src/riak_cs_config.erl]: `s/riak.conf/riak-cs.conf/`

Comment (shino) [inline src/riak_cs_config.erl]: No. I was wrong.
Cuttlefish uses schema and validate values. One can not set any value other than `boolean()`. If invalid value is set, advanced.config is probably wrong.

**Comment by shino** [comment]
Nice fix. Works as advertised. Only one fix is needed for log message before :+1:

**Comment by kuenishi** [comment]
Updated via web interface :P

**Comment by shino** [comment]
@borshop merge

**Comment by Basho-JIRA** [comment]
This is a bug made during 2.1 dev cycle, so release note for this is not needed. 

**_[posted via JIRA by Kota Uenishi]**_

**Comment by shino** [comment]
This PR was closed _BEFORE_ merge. Why?

**Comment by shino** [comment]
@borshop merge

**Comment by shino** [comment]
:confetti_ball:

**Pull request merged**